plugins/woocommerce/includes/rest-api/Controllers/Version3/class-wc-rest-paypal-webhooks-controller.php

```
@@ -0,0 +1,102 @@
+
+	/**
+	 * Test the webhook.
+	 *
+	 * phpcs:disable Generic.Commenting.Todo.TaskFound
+	 * TODO: Remove me before merging the feature branch.
+	 *
+	 * @param WP_REST_Request $request The request object.
+	 * @return WP_REST_Response The response object.
+	 */
+	public function test_webhook( WP_REST_Request $request ) {
+		$data = $request->get_json_params();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$data'. (Unused Code Rules)

(UnusedLocalVariable)